src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+* **Segment** consists of the name of the function and call segment being monitored.
+* **% Time** refers to the percentage of time each specific segment takes of the total transaction lifecycle. It's important to remember that some segments wait until all the other segments are completed before they're completed. In the screenshot below, the `Controller` segment is what is initiating the `POST` call, which means that all the other transactions are _nested_ within the `POST` call, which could be why it's taking longer than all the others. 
+* **Avg Calls/Txn** is the amount of times that the segment is called per transaction. 
+* **Avg Time** is the average time that a particular segment took to complete. It's important to keep in mind that there could be asynchronous work happening. One segment could be ongoing and then other, nested work, is being done at the same time. 
+
+Once you sort the **Breakdown table**, you can click on any problematic transactions. 
+
+<img
+    title="screenshot breakdown"
+    alt="A screenshot depicting the transaction breakdown "
+    src={apmTransactionBreakdown}
+/>
+</Step>
+
+<Step>
+### Explore transaction traces
+
+When your customers are trying to purchase something on your site and they use the `Pay Now` button, the `checkout-service` application generates multiple database queries, API calls, and other interactions with the underlying infrastructure. Transaction traces keep a record of every interaction between `checkout-service` and your customers. 
+
+Transaction traces can provide a wealth of information, including:
+
+    * The time spent on each segment of a transaction, such as database queries, external API calls, or rendering of HTML pages.
+    * The parameters passed to each segment of a transaction, allowing developers to identify specific data that may be causing issues.
+    * Information about errors or exceptions that occurred during a transaction.
+    * Information about the overall performance of the application, such as response times and throughput.
+
+
+<img
+    title="transaction traces in APM"
+    alt="A gif that displays transaction traces for a transaction"
+    src={apmTransactionTracesinAPM}
+/>
+
+</Step>
+
+<Step>
+### Take a look at trace details 
+
+<img
+    title="Trace details"
+    alt="A screenshot showing trace details for a transaction trace"
+    src={apmTraceDetails}
+/>
+
+</Step>
+<Step>
+And now you have all the information you need to fix the `Pay Now` button for your application. 
+
+* You know which transaction is causing the service degradation `Sinatra/Rubytron/POST #purchaseCart`.
+* You know when the problem started
+* You know which piece of the transaction is causing the breakdown in service. You and your team can now start fixing the origin of the problem and get your checkout up-and-running as quickly as possible. 
```

Comment:
I think the "you" in "You and your team" may be unnecessary here but that is just my thought looking at it. Totally a style choice and I just wanted to bring it up for consideration!

Reply:
I don't remember identifying which part of the transaction caused the breakdown in the service. Could you add a reference here?

Reply:
Good points, both of those!